packages/util/src/types.ts

```diff
@@ -20,11 +20,31 @@ export type BytesLike =
   | TransformabletoBytes
   | PrefixedHexString
 
+/**
+ * Byte type aliases for Uint8Array with a Byte length indicator
+ *
+ * Every type added here should have a corresponding PrefixedHex[NUM] type
+ * (see below).
+ */
+export type Bytes8 = Uint8Array
+export type Bytes20 = Uint8Array
+export type Bytes32 = Uint8Array
+
 /*
  * A type that represents a `0x`-prefixed hex string.
  */
 export type PrefixedHexString = string
 
+/**
+ * Prefixed hex string type aliases for Uint8Array with a Byte length indicator
+ *
+ * Every type added here should have a corresponding Bytes[NUM] type
+ * (see above).
+ */
+export type PrefixedHex8 = string
+export type PrefixedHex20 = string
+export type PrefixedHex32 = string
```

Comment:
Should these be updated to PrefixedHexString rather than string?

Reply:
Do not see the advantage of such alias chaining TBH.

Reply:
Ah you're right, but I was thinking in the context of that other PR (where PR was updated to `0x${string}`).